cli/bpmetadata/markdown.go

```diff
@@ -0,0 +1,99 @@
+package bpmetadata
+
+import (
+	"github.com/gomarkdown/markdown"
+	"github.com/gomarkdown/markdown/ast"
+)
+
+type mdContent struct {
+	literal   string
+	listItems []mdListItem
+}
+
+type mdListItem struct {
+	text string
+	url  string
+}
+
+// getMdContent accepts 3 types of content requests and return and mdContent object
+// with the relevant content info. The 3 scenarios are:
+// 1: get heading literal by (level and/or order) OR by title
+// 2: get paragraph content immediately following a heading by (level and/or order) OR by title
+// 3: get list item content immediately following a heading by (level and/or order) OR by title
+// A -1 value to headLevel/headOrder enforces the content to be matchd by headTitle
+func getMdContent(path []byte, headLevel int, headOrder int, headTitle string, getContent bool) *mdContent {
+	mdDocument := markdown.Parse(path, nil)
+	orderCtr := 0
+	if mdDocument != nil {
```

Comment:
Agreed with making this left aligned. This shouldn't be an error since it is possible that the readme may just not have that content available/configured e.g. missing `tagline`. If that's the case we just return `nil` and not add it to the metadata as opposed to raising an error.

Reply:
Since we always expect md contents, wouldn't this being nil signify something went wrong during parsing?